tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -153,6 +153,8 @@ class DatasetParams {
 
   virtual int op_version() const { return op_version_; }
 
+  virtual bool sloppy() const { return false; }
```

Comment:
can we move this to `GetNextTestCase` and call it `compare_order`? I'd like to avoid exposing dataset-specific options through `DatasetParams`.

Reply:
Good suggestion! Will change it.